modules/frontend/combiner/trace_by_id.go

```diff
@@ -156,6 +163,11 @@ func (c *traceByIDCombiner) shouldQuit() bool {
 		return false
 	}
 
+	// test special case for 413
+	if c.code == http.StatusRequestEntityTooLarge {
+		return false
```

Comment:
if the trace is too large, shouldn't we quit?

Reply:
I'm aligning with the comment here, which was counter to my intuition. 
 https://github.com/zalegrala/tempo/blob/0b86b085a23a07c16ed10e4137e349175c100ab0/modules/frontend/combiner/trace_by_id_test.go#L51

Reply:
yeah, ignore that test, we should quit 👍 

at this point the combiner has accepted as much as it will, by returning quit we cause the pipeline to stop farming out jobs to queriers and to return its results